### PR TITLE
feat(pagination): add numeric part

### PR DESCRIPTION
**Pull request by bennypowers**
## What I did

1. add a `numeric` part to the paginator

## Testing Instructions

1. check the dp https://deploy-preview-1338--red-hat-design-system.netlify.app/elements/pagination/demo/no-numeric-control/

## Notes to Reviewers

See #1298

### CODE REVIEW

File: elements/rh-pagination/demo/no-numeric-control.html [hunk 1]
```diff
@@ -0,0 +1,22 @@
+<rh-pagination>
```
Comment (brianferry) [inline elements/rh-pagination/demo/no-numeric-control.html:1]: Resolve when ready to merge

**Review by brianferry — CHANGES_REQUESTED** [review]

**Review by brianferry — APPROVED** [review]
Looks good to me, I am seeing the demo coming up correctly and the pagination part is hidden.  Default demo still functional.

File: elements/rh-pagination/demo/no-numeric-control.html (hunk 1 above)
Comment (bennypowers) [reply elements/rh-pagination/demo/no-numeric-control.html:1]: gonna pre-empt corey on this one, assuming he's cool with this change, as he brought it up originally

Comment (coreyvickery) [reply elements/rh-pagination/demo/no-numeric-control.html:1]: @bennypowers Looks good, thanks!

**Pull request merged**